doc/user/spatial_block_parameters.rst

```diff
@@ -0,0 +1,339 @@
+************************
+Spatial block parameters
+************************
+
+Many parameters assigned on a ``Block`` are scalar quantities that are useful for visualization
+and simple queries (e.g., block with the maximum burnup in an assembly). Spatial parameters in
+the a block, such as power produced by each pin, is also of interest. Especially when communicating
+data to physics codes that support sub-block geometric modeling. This page will talk about
+how spatial information is assigned to components on a block, how spatial data can be assigned
+and accessed, and how those data may or may not be updated by the framework.
+
+Sub-block spatial grid
+======================
+
+There are two ways to create the block grid: explicitly via blueprints, or via an automated builder. The former
+is recommended, but the later can work in some specific circumstances.
+
+Blueprints
+----------
+
+In your blueprints file, you likely have a core grid that defines where assemblies reside in the reactor. Assemblies
+are assigned to locations on that grid according to their ``specifcier`` blueprint attribute. Below is an example
+of a "flats up" hexagonal core grid of fuel assemblies with 1/3 symmetric.
+
+.. code:: yaml
+
+    grids:
+      core:
+        geom: hex
+        symmetry: third periodic
+        lattice map: |
```

Comment:
This is correct, but perhaps incomplete.

You can also define a lattice map as an explicit list of X/Y coordinates, and what lives there (if drawing pictures isn't your style):

https://github.com/terrapower/armi/blob/ffd6400a4902f7b5fb3188472d37e75e25f4a168/armi/tests/godiva/godiva-blueprints.yaml#L476-L484

I am fully aware this is not popular, but I'm just mentioning it.